plop-templates/storybook-boilerplate/component.core-template.ts.hbs

```diff
@@ -0,0 +1,13 @@
+import { {{ pascalCase name }} } from "dso-toolkit";
```

Comment:
Wat mij opvalt is dat hier na genereren van een nieuw component een foutmelding op verschijnt:

"TS2305: Module '"dso-toolkit"' has no exported member '<name>'."

Dit hangt wss samen met de 2 comments in plopfile.mjs:

// modify packages/dso-toolkit/src/index.ts with exports from new component

en

// modify storybook/src/templates.ts with new component

Heb een poging ondernomen om te kijken welke regels code waar moeten worden ge-append middels plop, maar dat is nog wel een beetje hogere plopkunde icm Storybook: storiesOf naar CSF...